ChristianChiarulli/LunarVim
# [Feature] Allow for custom path to a language server binary

# Description

Users will now be able to use a different language server binary. The path can be set in `lv-confg.lua`, e.g. 
`O.lang.lua.lsp_path = "path/to/different/binary"`

Some languages like dart (already has `sdk_path`) and java (it's a special child :) ) have been left as is.

## How Has This Been Tested?

- Confirm that the langauge server attaches by checking `:LspInfo`
- Add a custom path in `lv-config.lua` and confirm that the change is reflected
